asmcomp/amd64/emit.mlp

```diff
@@ -698,6 +698,12 @@ let emit_instr env fallthrough i =
   | Lop(Iintop_imm(op, n)) ->
       (* We have i.arg.(0) = i.res.(0) *)
       instr_for_intop op (int n) (res i 0)
+  | Lop(Icompf cmp) ->
+      let cond, need_swap = float_cond_and_need_swap cmp in
+      let a0, a1 = if need_swap then arg i 1, arg i 0 else arg i 0, arg i 1 in
+      I.cmpsd cond a1 a0;
+      I.movd a0 (res i 0);
```

Comment:
This should be `movq` not `movd` when argument and result are in 64-bit registers.

Reply:
I believe `movd` is right, strange as it sounds. But let me try to dig out a reference since I'm currently relying on some hazy memory.

Reply:
ok, thanks for checking.